tests/subsys/bluetooth/fast_pair/crypto/src/main.c

```diff
@@ -169,14 +170,136 @@ static void test_bloom_filter(void)
 			  "Invalid resulting filter.");
 }
 
+static void test_aes_ctr(void)
+{
+	if (!IS_ENABLED(CONFIG_BT_FAST_PAIR_TEST_CRYPTO_EXT)) {
+		ztest_test_skip();
+		return;
+	}
```

Comment:
This is no longer necessary. Let's test everything each time for each crypto backend (no need for another job in the testcase.yaml file). Same for other functions

Reply:
Ok, we can simplify it that way. @MarekPieta do you agree?